extensions/config/portability-config-yaml/src/main/java/org/dataportabilityproject/config/yaml/YamlSettingsExtension.java

```diff
@@ -79,4 +87,16 @@ void parse(InputStream in) {
       }
     }
   }
+
+  @VisibleForTesting
+  void parseRetryLibrary(InputStream in) {
+    if (in != null) {
+      ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+      try {
+        settings.put("retryLibrary", mapper.readValue(in, RetryStrategyLibrary.class));
```

Comment:
As a setting name it seems like this is the defaultRetrySettings/strategy and not just retryLibrary

Reply:
I wouldn't think that they would come from here though, I would have thought they would come from the extensions themselves.